Makefile

```diff
@@ -13,41 +7,67 @@ ifeq ($(TAGGED_VERSION),)
 endif
 VERSION ?= $(shell echo $(TAGGED_VERSION) | cut -c 2-)
 
-.PHONY: compare-deps
-compare-deps: Gopkg.lock GlooE-Gopkg.lock print-info
-	go run scripts/compare_dependencies.go Gopkg.lock GlooE-Gopkg.lock
+#----------------------------------------------------------------------------------
+# Retrieve GlooE build information
+#----------------------------------------------------------------------------------
+GLOOE_DIR := _glooe
+z := $(shell mkdir -p $(GLOOE_DIR))
```

Comment:
is `z` used anywhere?

Reply:
No, it's a quick way of creating the dir. Do you know if I can use `_`?

Reply:
It works, will change.